templates/insights/insights_page.html

```diff
@@ -5,10 +5,33 @@
 
 {% block content %}
     {% include 'includes/generic-intro--dark.html' %}
+    {% include 'includes/hero-img.html' %}
+    {% include 'includes/jumplinks.html' %}
 
     <div class="container">
+        {% include 'includes/section-separator.html' %}
+        {% include 'includes/content-block.html' %}
+        {% include 'includes/content-block--bordered.html' %}
+        {% include 'includes/content-block.html' %}
+
         {% for block in page.body %}
-            {% include_block block %}
+        {% include_block block %}
```

Comment:
A very minor point, but I might opt to keep this `include` statement indented to make it a bit easier to scan the code.